docs/ossm/README.md

```diff
@@ -6,4 +6,54 @@ This documentation is specific to the OpenShift Service Mesh product and may dif
 
 # Table of Contents
 
-TODO
+- [Running OSSM 2 and OSSM 3 side by side](#running-ossm-2-and-ossm-3-side-by-side)
+
+
+# Running OSSM 2 and OSSM 3 side by side
+This section describes how to run OSSM 2 and OSSM 3 side by side in one cluster without interfering with each other.
+It's not OSSM 2 -> OSSM 3 migration guide.
+
+To better understand steps described here, it's recommended to read OSSM 2 [deployment models](https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-deployment-models) and upstream Istio [Multiple Control Planes](https://istio.io/latest/docs/setup/install/multiple-controlplanes/) documentation. This document assumes OSSM 2.6 is already running and the goal is to install OSSM 3.0.
+
+## Cluster-Wide (Single Tenant) mesh deployment model
```

Comment:
We can add here a brief description like:
```
The starting point for this point is a OCP cluster with version 4.16 or higher and with Cluster-Wide single tenant depoyment model, please refer to this documentation to be sure that is your deployment type.
```

Reply:
We will support both multi and single tenant so we have chapters for both. Not sure if we need to include OCP version, it should be mentioned in support matrix.